src/components/task-step/exercise-multiple-choice.cjsx

```diff
@@ -110,13 +113,15 @@ ExerciseReview = React.createClass
     return step.has_recovery and step.correct_answer_id isnt step.answer_id
 
   renderFooterButtons: ->
-    isDisabledClass = 'disabled' unless @isContinueEnabled()
-    continueButton = <BS.Button bsStyle="primary" className={isDisabledClass} onClick={@onContinue}>Continue</BS.Button>
+    # TODO switch to using React.addons.classSet for classname
+    buttonClasses = '-continue'
+    buttonClasses += 'disabled' unless @isContinueEnabled()
+    continueButton = <BS.Button bsStyle="primary" className={buttonClasses} onClick={@onContinue}>Continue</BS.Button>
     tryAnotherButton = <BS.Button bsStyle="primary" onClick={@tryAnother}>Try Another</BS.Button> if @canTryAnother()
-    <span>
+    <div>
```

Comment:
I was getting an error during testing at some point about duplicate elements and one of the possible causes I found was doing funky stuff with HTML like putting a block element inside an inline element.  This ended up not being the problem so I can change this back?